google/cloud/bigtable/CMakeLists.txt

```diff
@@ -12,6 +12,91 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This should be included from the top level CMakeLists file
```

Comment:
Is the goal that each client library will have a separate version number, or that there will be a single version number for the repo, and all clients will be set to that version number in lock-step? If so, this should really be a TODO or FIXME with an issue id to ensure version correspondence.

Feel free to not block this PR on creating the issue and updating the comment, but this should be added here, so let's not forget about it.

Reply:
In other languages each client library gets its own version number **and** there is a version number for the overall project.  Doing the same thing for `google-cloud-cpp`.  Note that if we will use semver we need to have separate version numbers for each library or we will never get to 1.0.

Reply:
SGTM, thanks for the explanation.